verification/cmd/verification-service/README.md

```diff
@@ -21,6 +22,33 @@ configuration:
 
 The verifier currently doesn't support any configuration.
 
+### Session manager configuration
+
+Session manager has a single configuration point: `backend`. This specifies
+which `ISessionManager` implementation will be used. The following backends are
+supported:
+
+- `ttlcache`: the default; this creates the session cache in memory of the
+  `verification-service` process.
+- `memcached`: uses an external [memcached](https://www.memcached.org/) server.
+
+All other entries under `sessionmanager` must be backend names (i.e. `ttlcache`
+or `memcached`), providing backend-specific configuration. Only configuration
+for the backend selected by `backend` entry will actually be used.
+
+#### `ttlcache` backend
+
+`ttlcache` backend does not have any configuration points.
+
+#### `memcached` backend
+
+`memcached` backend has the following configuration points:
+
+- `servers` (optional): a list of servers in the "<host>:<port>" that will be
+  used by the backend. The servers will be used with equal weight. Adding the
+  same entry multiple times increases its weight. If this is not specified,
```

Comment:
interesting choice

Reply:
The choice was made by the `gomemcache` developer, I'm merely propagating it :)

Reply:
👍